### DISCUSSION TITLE
Getting parameter set in config.h

### DISCUSSION THREAD

**Discussion by cpapplefamily** [grblHAL/core#366] [Q&A]
For instance, I wish to have $32 = 1 but when I try to build the file and look at the results in my gcode sender it remains = 0

/*! @name $32 - Setting_Mode
__NOTE:__ only one mode can be enabled.
 */
///@{
#if !defined DEFAULT_LASER_MODE || defined __DOXYGEN__
#define DEFAULT_LASER_MODE On
#endif
#if !defined DEFAULT_LATHE_MODE || defined __DOXYGEN__
#define DEFAULT_LATHE_MODE Off
#endif
///@}

**Comment by terjeio** [comment]
Even after [resetting settings](https://github.com/grblHAL/core/blob/c3fd2db2e26fccc779b4721d590310450528bf66/config.h#L26-L29)?

**Comment by cpapplefamily** [comment]
I bet this is the reason - RTFM

`A reset of non-volatile storage with `$RST=*` after reflashing is also required.`
